pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -9,6 +9,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/appscode/jsonpatch"
```

Comment:
This repo seems not to be actively maintained. Do you see us maintaining it in the future?

Reply:
Good point. So far I'm checking if the general approach fixes the problem; might switch to something else.